lib/pbench/server/database/models/datasets.py

```diff
@@ -253,7 +254,7 @@ class Dataset(Database.Base):
 
     Columns:
         id          Generated unique ID of table row
-        owner_id    Owning UUID of the owner of the dataset
+        owner_id    User_id of the owner of the dataset
```

Comment:
Sigh; I think the old "Owning UUID" would be better than the completely undefined "User_id" tag here.

And although it's not a *column*, describing the `owner` relationship as an *argument* wouldn't be a terrible idea.

But those are trivia.